lib/bitvec.cpp

```diff
@@ -36,6 +36,7 @@ std::ostream &operator<<(std::ostream &os, const bitvec &bv) {
 }
 
 bitvec &bitvec::operator>>=(size_t count) {
+    assert(static_cast<int>(count) >= 0);
```

Comment:
Yeah, it's an unsigned. I wonder though if we should set a more constrained limit on the size of bitvec. And we should assert that the return from new in the 'expand' routine is non-null.